### PR TITLE
new command select

### PR DESCRIPTION
**Pull request by KacperBorowiecki**

### CODE REVIEW

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs [hunk 1]
```diff
@@ -313,6 +313,39 @@ public void AlertPerformAction(string action, TimeSpan timeout)
             }
         }
 
+        public void Select(string selectValue, string selectBy, SeleniumCommandArguments search, TimeSpan timeout)
+        {
+            var element = GetElementInFrame(search, timeout);
+            var select = new SelectElement(element);
+
+            if (selectBy.ToLower()=="index")
```
Comment (pwitusik) [inline G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: switch from "if" to "switch"

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs [hunk 2]
```diff
@@ -313,6 +313,39 @@ public void AlertPerformAction(string action, TimeSpan timeout)
...
+                try
+                {
+                    int index = int.Parse(selectValue);
+                }
+                catch (Exception)
+                {
+
+                    throw new ApplicationException($"{selectValue} is not an integer number");
+                }
+                select.SelectByIndex(int.Parse(selectValue));
```
Comment (pwitusik) [inline G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: move this line into the try block

**Review by pwitusik — CHANGES_REQUESTED** [review]

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs (hunk 2 above)
Comment (KacperBorowiecki) [reply G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: I did it in another way  due to information of parsing error

File: G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs [hunk 3]
```diff
@@ -313,6 +313,41 @@ public void AlertPerformAction(string action, TimeSpan timeout)
...
+
+                    try
+                    {
+                        index = int.Parse(selectValue);
+                    }
+                    catch (Exception)
+                    {
+                        throw new ApplicationException($"{selectValue} is not an integer number");
+                    }
+                    select.SelectByIndex(index);
```
Comment (pwitusik) [inline G1ANT.Addon.Selenium/Api/SeleniumWrapper.cs]: move to try block

**Review by pwitusik — CHANGES_REQUESTED** [review]

**Review by pwitusik — APPROVED** [review]

**Pull request merged**